contexts/index.tsx

```diff
@@ -5,20 +5,21 @@ import React from 'react';
 
 import BaseUrlContext from './BaseUrl';
 import GlossaryContext from './GlossaryTerms';
+import ThemeContext from './Theme';
 
-type Props = Pick<RunOpts, 'baseUrl' | 'terms' | 'variables'> & React.PropsWithChildren;
+type Props = Pick<RunOpts, 'baseUrl' | 'terms' | 'theme' | 'variables'> & React.PropsWithChildren;
```

Comment:
Maybe move the prop `theme` from `RunOpts` to some other type????

```suggestion
type Props = Pick<RunOpts, 'baseUrl' | 'terms' | 'variables'> & React.PropsWithChildren & RMDXModuleProps;
```

*types.d.ts*
```javascript
type RMDXModuleProps = {
  theme: 'light' | 'dark';
}
```